code/src/components/Question2/Question2.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+import './Question2.css';
+
+const lostItems = ['suitcase', 'mind', 'dog', 'energy'];
```

Comment:
Great that you created an array to map through instead of repeating the same piece of code below multiple times. Easier to add of remove items in it as well.